javascript/organizing_your_javascript_code/project_restaurant_page.md

```diff
@@ -16,24 +16,20 @@ Let's use what we've learned and take a chance to continue practicing DOM manipu
 
     1. Create a `src` and `dist` directory with the following contents:
-        1. an `index.js` file in `src`
+        1. an `index.js` file in `src`.
 
-        1. an `index.html` file in `dist`. Go ahead and link the `main.js` file in a script tag. `main.js` is the file that will be generated by webpack.
+        1. an `index.html` file in `src`. This file will not need a script tag, because we're using `html-webpack-plugin`, which automatically adds that in. You will also not need to link a CSS stylesheet as you should be importing it into your JavaScript and letting your webpack configuration handle bundling.
 
-        1. create a `webpack.config.js` file that looks just like our file from the [tutorial](https://webpack.js.org/guides/getting-started/#using-a-configuration).
+        1. create a `webpack.config.js` file that looks just like our file from the [tutorial](https://webpack.js.org/guides/getting-started/#using-a-configuration). Don't forget to add the `html-webpack-plugin` config to your `webpack.config.js` and set its `template` option with a path to `src/index.html`.
 
-1. Set up an HTML skeleton inside of `dist/index.html`. Inside the body, add a `<header>` element that contains a `<nav>` with buttons (not links!) for different "tabs" (for example buttons for "Home", "Menu" or "About" etc). Below the `<header>`, add a single `<div id="content">`.
+1. Set up an HTML skeleton inside of `src/index.html`. Inside the body, add a `<header>` element that contains a `<nav>` with buttons (not links!) for different "tabs" (for example buttons for "Home", "Menu" or "About" etc). Below the `<header>`, add a single `<div id="content">`.
 
-1. Inside of `src/index.js` write a console.log or alert statement and then run `npx webpack`. Load up `dist/index.html` in a browser to make sure everything is working correctly.
+1. Inside of `src/index.js` write a `console.log` or `alert` statement and then run `npx webpack`. Load up `src/index.html` in a browser to make sure everything is working correctly.
 
     - Quick tip #2: if you run `npx webpack --watch` you will not have to rerun webpack every time you make a change.
 
 1. Inside `div#content`, create a homepage for your restaurant. You might want to include an image, headline, and some text about how wonderful the restaurant is; you do not have to make this look too fancy. It’s okay to hard-code these into the HTML for now just to see how they look on the page.
 
-1. Since we're using the `Html-Webpack-Plugin`, you will need to edit your `src/index.js` file to insert the `div#content` into the body; `Html-Webpack-Plugin` creates a new HTML page which will remove your `src/index.html` file.
```

Comment:
```
1. Now remove everything inside `div#content` from the HTML (so you still have the `<header>` and `<nav>` with an empty `<div id="content">` below it) and instead create them by using JavaScript only, e.g. by appending each new element to `div#content` once the page is first loaded. Since we're all set up to write our code in multiple files, let's write this initial page-load function inside of its own module and then import and call it inside of `index.js`.
```
Could you pop the above line back in here please so we get the original step back?
You can view the [current website's project instructions](https://www.theodinproject.com/lessons/node-path-javascript-restaurant-page) to make sure all the instruction steps are included in the right order.

Reply:
Done